basilcea/politico
# #163393702  Setup server side framework

#### What does this PR do?
Merge server side framework to develop branch
#### Description of Task to be completed?
Setup Express framework in Node Js for creating API endpionts
#### How should this be manually tested?
To check if node js and NPM is properly installed, type node -v and npm -v respectively on terminal or command prompt
check package.json to ensure the installed dependencies are there eithe as  dependencies or devDependencies
#### What are the relevant pivotal tracker stories?
Pivotal tacker Id - #163393702
